script.js

```diff
@@ -1,4 +1,66 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // 1 = Scissors, 2 = Paper, 3 = Stone
+  var randomAction = randomSPS();
+  myOutputValue =
+    'Error. Please input only "Scissors", "Paper" or "Stone". Try agin!';
+  if (randomAction == 1) {
+    actionThrown = "Scissors";
+  }
+  if (randomAction == 2) {
+    actionThrown = "Paper";
+  }
+  if (randomAction == 3) {
+    actionThrown = "Stone";
+  }
```

Comment:
Try to make another function for this. 
E.g. a separate  function that returns scissors paper or stone.